heat/core/types.py

```diff
@@ -539,6 +540,29 @@ def can_cast(from_, to, casting="intuitive"):
                 break
 
 
+def issubdtype(arg1, arg2):
+    """
+    Returns True if first argument is a typecode lower/equal in type hierarchy.
+
+    Parameters
+    ----------
+    arg1 : type, str, ht.dtype
+        A description representing the type. It may be a a Python builtin type, string or an HeAT type already.
+    arg2 : type, str, ht.dtype
+        A description representing the type. It may be a a Python builtin type, string or an HeAT type already.
+
+    Returns
+    -------
+    out : bool
+        Result of the stated comparison
+    """
```

Comment:
all looks good. can you add an example or two?

Reply:
Glad to hear - Yes, for sure.